src/pudl_archiver/archivers/classes.py

```diff
@@ -33,6 +34,19 @@
 }
 VALID_PARTITION_RANGES: dict[str, str] = {"year_quarter": "QS", "year_month": "MS"}
 
+USER_AGENTS: list[str] = [
```

Comment:
registering slight approval of this rotation approach in a github comment for posterity

Reply:
@e-belfer can you explain a bit about how you compiled these guys? in case someone in the future wants to add more or change em?